pages/database-management/experimental-features.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Experimental features
+description: Experimental features are new functionalities in Memgraph that carry some risk. They are still in development and may have stability issues.
```

Comment:
Please add a phrase along the lines of "contact your support representative when encountering any challenges and we'll respond promptly to get you back on track quickly"